server/app/services/geo/esri/EsriClient.java

```diff
@@ -30,6 +31,12 @@ public abstract class EsriClient {
 
   private final Logger logger = LoggerFactory.getLogger(this.getClass());
 
+  private static final Histogram esriLookupTime =
```

Comment:
**nitpick:** use SCREAMING_SNAKE_CASE for constants

Reply:
Done